### PR TITLE
Bonus guide: Bisq

### PR DESCRIPTION
**Pull request by VajraOfIndra**
#### What

This is a proposal for a new bonus guide that explains how to install Bisq on a personal computer and connect it to one's own Bitcoin node via Tor onion address for maximum privacy.

### Why

Connecting Bisq to a remote node within the same LAN can be quite tricky as using <nodeIP>:8333 does not seem to work (e.g., see issue [here](https://www.reddit.com/r/bisq/comments/pikpxq/cant_connect_to_my_remote_raspibolt_node/)). Using the onion address solves the problem but requires digging up for it which is not straightforward. Therefore, a guide is judged necessary to avoid users the recommended Bisq nodes.

#### How

* Created a `bisq.md` guide in `/bonus/+bitcoin`
* Modified the +bitcoin `index.md` to add an entry for Bisq under the Privacy heading (is that a OK classification for Bisq?)

The guide focuses on:
1. Solving the connection to one own's node
2. Configuring Bisq for privacy and security.

* For the installation and verification, it points to Bisq informative webpage.
* The guide does not explain how to set up a local currency account and does not explain how to make a trade. The user is redirected to Bisq website and wiki to find that out. 
The guide could be expanded later on to add these two sections if it's judged necessary or is asked for by users.

#### Scope

- [ ] significant change to core configuration
- [x] independent bonus guide
- [ ] simple bug fix

#### Test & maintenance

Test: Follow the guide
Maintenance: Occasional check up for UI changes. Occasional checking for dead links.

#### Animated GIF (optional)

rollercoaster

### CODE REVIEW

File: guide/bonus/bitcoin/bisq.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+### Bitcoin Explorer
+
+* Click to "Settings" > "Preferences"
+
+* Click on the three dots of the "Bitcoin Explorer" section
+
+* Use the following information:
+  * Name: Choose a name (e.g., `RaspiBolt`)
+  * Transaction URL: `https://raspibolt.local:4000/tx/` (replace rapsibolt.local by your node IP address if required)
```
Comment (Stadicus) [inline guide/bonus/bitcoin/bisq.md:140]: s/rapsibolt/raspibolt/ :)
s/by/with/

File: guide/bonus/bitcoin/bisq.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+  * Address URL: `https://raspibolt.local:4000/address/` (replace rapsibolt.local by your node IP address if required)
```
Comment (Stadicus) [inline guide/bonus/bitcoin/bisq.md:141]: ^-- same

**Review by Stadicus — APPROVED** [review]
Nice addition. I did not work through the guide, it's more of a visual check.

utACK

File: guide/bonus/bitcoin/bisq.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+
+#### Start Bisq from the command line
+
+The first time Bisq is opened using the GUI, it will connect to several remote Bitcoin nodes via Tor. 
+To avoid this, we will start Bisq the first time using the command line to force it to connect to your Bitcoin node only.
+
+* On your personal computer where you installed Bisq, open a command line terminal
+
+* We will start Bisq with two flags that will force it to connect to Tor only and using our own Bitcoin Core onion address. 
+Replace `123...abc.onion:8333` by your own Bitcoin Core .onion address that you obtained above.
```
[the remark this answers is not in the document]
Comment (VajraOfIndra) [reply guide/bonus/bitcoin/bisq.md:96]: fixed

File: guide/bonus/bitcoin/bisq.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+
+* Click on the "Settings" tab
+
+* Click on the "Network info" tab
+
+* In the "Bitcoin Network" section, click on "Use custom Bitcoin Core nodes"
+
+* In the box just below, paste your Bitcoin Core node .onion address (`123...abc.onion:8333`)
+
+* Click on any other tab at the top. Bisq will ask you to shut down the program to make your change effectice.
```
[the remark this answers is not in the document]
Comment (VajraOfIndra) [reply guide/bonus/bitcoin/bisq.md:112]: fixed

File: guide/bonus/bitcoin/bisq.md (hunk 1 above)
Comment (VajraOfIndra) [reply guide/bonus/bitcoin/bisq.md:140]: fixed :)

File: guide/bonus/bitcoin/bisq.md (hunk 2 above)
Comment (VajraOfIndra) [reply guide/bonus/bitcoin/bisq.md]: fixed

**Comment by VajraOfIndra** [comment]
> Nice addition. I did not work through the guide, it's more of a visual check.
> 
> utACK

Thanks, I fixed the bugs you pointed out :+1:

**Pull request merged**